python/hsfs/connection.py

```diff
@@ -153,8 +154,8 @@ def get_feature_store(self, name: str = None):
             `FeatureStore`. A feature store handle object to perform operations on.
```

Comment:
@kennethmhc  - can you please update the documentation making it clear that we just need the project name?

Reply:
@SirOibaf Changed the documentation.